x-pack/plugins/security_solution/common/api/timeline/get_timelines/get_timelines_route.gen.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+/*
+ * NOTICE: Do not edit this file manually.
+ * This file is automatically generated by the OpenAPI Generator, @kbn/openapi-generator.
+ *
+ * info:
+ *   title: Elastic Security - Timeline - Get Timelines API
+ *   version: 2023-10-31
+ */
+
+import { z } from 'zod';
+
+import {
+  TimelineType,
+  SortFieldTimeline,
+  TimelineStatus,
+  TimelineResponse,
+} from '../model/components.gen';
+
+export type GetTimelinesRequestQuery = z.infer<typeof GetTimelinesRequestQuery>;
+export const GetTimelinesRequestQuery = z.object({
+  /**
+   * If true, only timelines that are marked as favorites by the user are returned.
+   */
+  only_user_favorite: z.enum(['true', 'false']).nullable().optional(),
+  timeline_type: TimelineType.nullable().optional(),
+  sort_field: SortFieldTimeline.optional(),
```

Comment:
I missed this in mine, but `sort_field` and `sort_order` aren't optional

Reply:
@michaelolo24 The query schema is defined as `partial`:

```typescript
export const getTimelinesQuerySchema = rt.partial({
  ...
  sort_field: sortFieldTimeline,
  sort_order: direction,
  ...
});
```

https://github.com/elastic/kibana/blob/a51b775391af09b6ba2ee9456489ef3351f8875a/x-pack/plugins/security_solution/common/api/timeline/get_timelines/get_timelines_route.ts#L24-L25

Reply:
Oops, thanks! Yep :)